Code/Framework/AzCore/AzCore/Math/Guid.h

```diff
@@ -66,9 +66,9 @@ typedef const GUID& REFIID;
 const GUID name \
 = { l, w1, w2, { b1, b2,  b3,  b4,  b5,  b6,  b7,  b8 } }
 
-static REFGUID GUID_NULL()
+inline static REFGUID GUID_NULL()
```

Comment:
static isn't needed with inline for free functions
That makes it so that each translation unit that includes Guid.h gets it's inline version of the function due to `static` making the function have internal linkage instead of external linkage by default

Reply:
without it clang reports that the function is not used in some translation units

Reply:
let me try, I just added the inline because Clang reports that the function is unused in some compilation units